prov/verbs/src/fi_verbs.c

```diff
@@ -60,10 +60,14 @@ struct fi_provider fi_ibv_prov = {
 	.cleanup = fi_ibv_fini
 };
 
-struct util_prov fi_ibv_util_prov = {
-	.prov = &fi_ibv_prov,
-	.info = NULL,
-	.flags = UTIL_RX_SHARED_CTX,
+struct fi_ibv_infra fi_ibv_verbs_infra = {
```

Comment:
What's 'infra' stand for?

Reply:
'infra' stands for of infrastructure. I'm not sure that is proper name for that.
But the `fi_ibv_verbs_infra` contains the list of the verbs devices